src/mlpack/config.hpp

```diff
@@ -72,6 +72,15 @@
   #define MLPACK_CERR_STREAM std::cerr
 #endif
 
+
+//
+// MLPACK_NO_STD_MUTEX is used to disable mutex usage inside mlpack. Assuming
+// the system has one core only.
+//
+#ifdef MLPACK_NO_STD_MUTEX
+  #define ARMA_DO_NOT_USE_STD_MUTEX
```

Comment:
@shrit @rcurtin This doesn't look right.  The correct option is `ARMA_DONT_USE_STD_MUTEX`, not `ARMA_DO_NOT_USE_STD_MUTEX`.

https://arma.sourceforge.net/docs.html#config_hpp

Reply:
Nice catch!  I will make a quick PR for this tomorrow (unless you beat me to it) then rebase the release.